mozilla/fx-private-relay
# MPP-3528: Recreate website metrics, add images

This rewrites `METRICS.md` to include all the frontend code that emits Google Analytics events, as well as sample images to make it easier to refer to buttons and text.

The images are minimized for size with [pngquant](https://pngquant.org/). They are displayed at 1/3 width, and can be clicked to see the full image.

# How to test:

* Read the doc, see if anything sounds off. Don't feel you have to read all the details, skimming is fine.
* Spot check by searching for categories, labels, etc in `frontend/src`.
